tests/unit/components/DocumentationTopic.spec.js

```diff
@@ -347,21 +338,17 @@ describe('DocumentationTopic', () => {
       expect(wrapper.contains(RequirementMetadata)).toBe(true);
     });
 
-    it('hides the Summary, if the global settings say so', () => {
-      // this should really only mock the resolved value for the specific flag,
-      // but this is fine for now
-      getSetting.mockResolvedValueOnce(true);
-      wrapper = shallowMount(DocumentationTopic, { propsData });
-      expect(wrapper.find(Summary).exists()).toBe(false);
-    });
-
     it('renders a `Availability` with platforms data', () => {
       const list = wrapper.find(Availability);
       expect(list.exists()).toBe(true);
       expect(list.props('platforms')).toEqual(propsData.platforms);
     });
+  });
 
+  describe('isTargetIDE', () => {
```

Comment:
You could have just specified it in the name of the test. No need to wrap it in a describe, if its only 1?